src/py/covidr/config/development.py

```diff
@@ -15,3 +15,8 @@ class DevelopmentDatabaseConfig(DatabaseConfig):
     @property
     def URI(self):
         return "postgresql://user_rw:password_rw@localhost:5432/covidr_db"
+
+    @property
+    def SEND_FILE_MAX_AGE_DEFAULT(self):
+        """Ensures that latest static assets are read during frontend dev work."""
+        return 0
```

Comment:
interesting, i thought the static assets would be overwritten when recompiled.

Reply:
They are, but that doesn't necessarily mean Flask will send them instead of telling the browser to use it's cache.

Reply:
oh cool makes sense, thanks for explaining